spyder/plugins/editor/widgets/codeeditor.py

```diff
@@ -3064,6 +3088,40 @@ def visible_blocks(self):
     def is_editor(self):
         return True
 
+    def popup_docstring(self, prev_text, prev_pos):
+        """Show the menu for generating docstring"""
+        line_text = self.textCursor().block().text()
+        if line_text != prev_text:
+            return
+
+        if prev_pos != self.textCursor().position():
+            return
+
+        writer = self.writer_docstring
+        if writer.get_function_definition_from_below_last_line():
+            point = self.cursorRect().bottomRight()
+            point = self.calculate_real_position(point)
+            point = self.mapToGlobal(point)
+
+            self.menu_docstring = QMenuOnlyForEnter(self)
+            self.docstring_action = create_action(
+                self, _("Write docstring"), icon=ima.icon('TextFileIcon'),

```

Comment:
Again, like in the context menu, this should be ``Generate docstring`` for clarity and consistency.